### DISCUSSION TITLE
Cannot load explore tab

### DISCUSSION THREAD

**Discussion by Satont** [community/community#31558] [Mobile]
When opening explore tab I'm getting error. I cannot copy if from this message, you doesnt provide way for it. 

![Screenshot_20220901-041836_GitHub.png](https://user-images.githubusercontent.com/42675886/187812232-3c5f4b11-64d1-4eed-b609-50a5c61b7f39.png)

**Comment by candyho** [comment]
Hey, thanks for reporting! Do you mind sharing with us which mobile device are you using, your OS and the app version number? That'd definitely help us to speed up the investigation.

**Reply by Satont** [reply]
App: 1.78.0
Android: 12

Also i cleared cache and now this tab works, but I'm not sure about how to reproduce it.
